index.html

```diff
@@ -477,7 +477,7 @@ <h3>Character</h3>
 
             <li>If the <a>Character</a> has a <a>Talent Name</a> property, the following TTML constraints apply:
               <ul>
-                <li>Another <code>ttm:agent</code> MUST be added to the <code>metadata</code> element that indicates the <a>Script Type</a> in the <code>head</code> element, with the following:
+                <li>Another <code>ttm:agent</code> MUST be added to a <code>metadata</code> element in the <code>head</code> element, with the following:
```

Comment:
Equivalent change to proposal above, plus add a missing word "constraints":

```suggestion
            <li>Another <code>&lt;ttm:agent&gt;</code> element MUST be added at the path
              <code>/tt/head/metadata/ttm:agent</code>, with the following constraints:
```